submissions/7SergSmith7/HTML-CSS-Popup/styles.css

```diff
@@ -0,0 +1,144 @@
+a {
+  text-decoration: none;
+  color: black;
+}
+a:focus {
+  color: gray;
+  border: 1px solid gray;
+}
+a:hover {
+  color: gray;
+  border: 1px solid gray;
+}
+.right_navigation__mail,
+.right_navigation__images {
+  align-self: center;
+}
+.popup_btn-icon:focus,
+.show-more:focus {
+  color: gray;
+  border: 1px solid gray;
+}
+.popup_btn-icon:hover,
+.show-more:hover {
+  color: gray;
+  border: 1px solid gray;
+}
+.containers {
+  width: 1000px;
+}
+label:focus {
+  outline-style: none;
+  color: gray;
+  border: 1px solid gray;
+}
+.right_navigation {
+  width: 250px;
+  float: right;
+  display: flex;
+  justify-content: space-between;
+}
+.right_navigation img {
+  width: 25px;
+}
+
+.right_navigation__avatar img {
+  border-radius: 50%;
+}
+.popup_btn,
+.show-more_btn {
+  display: none;
+}
+
+.popup_btn:checked ~ .popup {
```

Comment:
You can't open popup using only keyboard.
This popup must be opened by mouse or keyboard.